tests/components/synology_dsm/test_config_flow.py

```diff
@@ -48,9 +49,10 @@ def mock_controller_service():
         "homeassistant.components.synology_dsm.config_flow.SynologyDSM"
     ) as service_mock:
         service_mock.return_value.login = Mock(return_value=True)
```

Comment:
```suggestion
        service_mock.return_value.login = True
```
?

Reply:
Looks like `login` is a method that is called? So it needs to be its own Mock with a return value.

Reply:
Mocking tests and I are two separate things for now :smile: